metadata.yml

```diff
@@ -19,7 +19,7 @@ annotations:
   io.kubewarden.policy.ociUrl: ghcr.io/kubewarden/policies/persistentvolumeclaim-storageclass-policy
   # kubewarden specific:
   io.kubewarden.policy.title: persistentvolumeclaim-storageclass-policy
-  io.kubewarden.policy.version: 1.0.4
+  io.kubewarden.policy.version: 1.2.0
```

Comment:
why not 1.1.0? the latest release is 1.0.4

Reply:
Ops, just a typo. I'll remove the bump and rely on the automation to do the release.